airbyte-integrations/connectors/destination-s3-glue/src/main/resources/spec.json

```diff
@@ -132,19 +132,70 @@
                 "description": "Whether the input json data should be normalized (flattened) in the output JSON Lines. Please refer to docs for details.",
                 "default": "Root level flattening",
                 "enum": ["No flattening", "Root level flattening"]
+              },
+              "stringify": {
+                "type": "string",
+                "title": "Stringify JSON",
+                "description": "Whether nested objects in the input json data should be converted to strings in the output JSON Lines. Please refer to docs for details.",
+                "default": "Default",
+                "enum": ["Default", "Stringify"]
               }
             }
           }
         ],
         "order": 5
       },
+      "numeric": {
+        "title": "Numeric Type",
+        "type": "object",
+        "description": "Hive compatible type to map fractional numeric values. See <a href=\"https://cwiki.apache.org/confluence/display/hive/languagemanual+types#LanguageManualTypes-NumericTypes\">here</a> for more details",
+        "oneOf": [
+          {
+            "title": "Double",
```

Comment:
Should Float be an option, too?

Reply:
I think there may be a case where float would be desired, but Double is essentially the same with a greater scale, so I think keeping it as double or decimal is a good starting point.